app/plugins/Wallet/js/loaders.js

```diff
@@ -74,11 +73,11 @@ IPC.on('unlocked', function(err, result) {
 });
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Encrypting ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-// setUnencrypted sets the wallet lock status to unencrypted.
+// setUnencrypted sets the wallet lock status to encrypted.
 function setUnencrypted() {
 	clearLockIcon();
-	eID('lock-status').innerHTML = 'Create Wallet';
-	eID('lock-icon').classList.add('fa-plus');
+	eID('lock-status').innerHTML = 'Unencrypted';
```

Comment:
Reverted `setUnencrypted` to original in f4c4c08ca1012ec5fb17baeb1d917adeb65c2620